<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>滴滴出行模拟系统</string>
  </property>
  <property name="styleSheet">
   <string>QMainWindow {
    background-color: #f5f7fa;
    border: none;
   }</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="titleLabel">
      <property name="text">
       <string>滴滴出行模拟系统</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="font">
       <font>
        <pointsize>32</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string>QLabel {
        color: #00b2ff;
        text-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
       }</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="buttonWidget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="passengerBtn">
         <property name="text">
          <string>乘客端</string>
         </property>
         <property name="minimumSize">
          <size>
           <width>220</width>
           <height>180</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>18</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string>QPushButton {
           background-color: #00b2ff;
           color: white;
           border-radius: 15px;
           box-shadow: 0 4px 12px rgba(0, 178, 255, 0.3);
           border: none;
          }
          QPushButton:hover {
           background-color: #0099cc;
           box-shadow: 0 6px 16px rgba(0, 178, 255, 0.4);
           transform: translateY(-2px);
          }
          QPushButton:pressed {
           background-color: #0088b3;
           box-shadow: 0 2px 8px rgba(0, 178, 255, 0.2);
           transform: translateY(0);
          }</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="driverBtn">
         <property name="text">
          <string>司机端</string>
         </property>
         <property name="minimumSize">
          <size>
           <width>220</width>
           <height>180</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>18</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string>QPushButton {
           background-color: #52c41a;
           color: white;
           border-radius: 15px;
           box-shadow: 0 4px 12px rgba(82, 196, 26, 0.3);
           border: none;
          }
          QPushButton:hover {
           background-color: #43a047;
           box-shadow: 0 6px 16px rgba(82, 196, 26, 0.4);
           transform: translateY(-2px);
          }
          QPushButton:pressed {
           background-color: #388e3c;
           box-shadow: 0 2px 8px rgba(82, 196, 26, 0.2);
           transform: translateY(0);
          }</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="platformBtn">
         <property name="text">
          <string>平台监控端</string>
         </property>
         <property name="minimumSize">
          <size>
           <width>220</width>
           <height>180</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>18</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string>QPushButton {
           background-color: #fa8c16;
           color: white;
           border-radius: 15px;
           box-shadow: 0 4px 12px rgba(250, 140, 22, 0.3);
           border: none;
          }
          QPushButton:hover {
           background-color: #f57c00;
           box-shadow: 0 6px 16px rgba(250, 140, 22, 0.4);
           transform: translateY(-2px);
          }
          QPushButton:pressed {
           background-color: #ef6c00;
           box-shadow: 0 2px 8px rgba(250, 140, 22, 0.2);
           transform: translateY(0);
          }</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="infoLabel">
      <property name="text">
       <string>请选择您的角色进入相应功能界面</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string>QLabel {
        color: #666666;
        font-weight: 500;
       }</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="registerWidget" native="true">
      <property name="styleSheet">
       <string>QWidget {
        background-color: transparent;
       }</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="registerPassengerBtn">
         <property name="text">
          <string>注册乘客</string>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string>QPushButton {
           background-color: #52c41a;
           color: white;
           border-radius: 8px;
           padding: 8px 20px;
           box-shadow: 0 2px 4px rgba(82, 196, 26, 0.2);
           border: none;
          }
          QPushButton:hover {
           background-color: #43a047;
           box-shadow: 0 3px 6px rgba(82, 196, 26, 0.3);
          }
          QPushButton:pressed {
           background-color: #388e3c;
           box-shadow: 0 1px 2px rgba(82, 196, 26, 0.2);
          }</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="registerDriverBtn">
         <property name="text">
          <string>注册司机</string>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string>QPushButton {
           background-color: #fa8c16;
           color: white;
           border-radius: 8px;
           padding: 8px 20px;
           box-shadow: 0 2px 4px rgba(250, 140, 22, 0.2);
           border: none;
          }
          QPushButton:hover {
           background-color: #f57c00;
           box-shadow: 0 3px 6px rgba(250, 140, 22, 0.3);
          }
          QPushButton:pressed {
           background-color: #ef6c00;
           box-shadow: 0 1px 2px rgba(250, 140, 22, 0.2);
          }</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
